src/pybind/mgr/dashboard_v2/settings.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import errno
+import inspect
+from six import with_metaclass
+
+from . import mgr
+
+
+class Options(object):
+    """
+    If you need to store some configuration value please add the config option
+    name as a class attribute to this class.
+    """
+    # Grafana settings
+    GRAFANA_API_HOST = ('localhost', str)
+    GRAFANA_API_PORT = (3000, int)
+    GRAFANA_API_USERNAME = ('admin', str)
+    GRAFANA_API_PASSWORD = ('admin', str)
+    GRAFANA_API_SCHEME = ('http', str)
+
+    # RGW settings
+    RGW_API_HOST = ('', str)
```

Comment:
I think last time we were looking at similar code, there was some discussion of getting the RGW settings automatically so that not manual config was needed -- anybody been working in that area?  @mattbenjamin ?

Reply:
@jcsp do you mean that RGW daemons report those configs (HOST, PORT ...) to ceph-mgr as daemon's metadata?

Reply:
This setting will be used by the RGW proxy (in the works, no PR yet) to override the automatic detection of a host and port. The RGW proxy will use the service map to get the host and port, but I'm not 100% sure that this is correct for most setups, but, AFAIK, it's not possible to get more data as a Ceph MGR module (like the data available through `radosgw-admin`).

Reply:
Actually, I'm going to remove all the options from this PR. These options should be added by the PR that needs them. This PR should only add the infrastructure.